src/main/java/com/jvavateam/carsharingapp/controller/RentalController.java

```diff
@@ -26,23 +31,47 @@
 public class RentalController {
     private final RentalService rentalService;
 
+    @PostMapping("/manager")
+    @PreAuthorize("hasRole('ROLE_MANAGER')")
```

Comment:
```suggestion
    @PreAuthorize("hasAuthority('MANAGER')")
``` do it with all roles